ci-operator/step-registry/upi/deprovision/heterogeneous/powervs/upi-deprovision-heterogeneous-powervs-ref.yaml

```diff
@@ -0,0 +1,12 @@
+ref:
+  as: upi-deprovision-heterogeneous-powervs
+  from: upi-installer
+  grace_period: 10m
+  commands: upi-deprovision-heterogeneous-powervs-commands.sh
+  resources:
+    requests:
+      cpu: 10m
+      memory: 100Mi
+  documentation: >-
+    This step deprovisions the powervs nodes created by upi-install-heterogeneous-powervs.
+    It requires the ipi-deprovision step already be executed against the cluster.
```

Comment:
Actually, this should run first, then the ipi-deprovision. The reason is there are networking elements setup in the VPC.